src/encoded/search.py

```diff
@@ -582,7 +582,7 @@ def format_facets(es_results, facets, used_filters, schemas, total, principals):
         if agg_name not in aggregations:
             continue
         terms = aggregations[agg_name][agg_name]['buckets']
-        if not any(bucket['doc_count'] > 0 for bucket in terms if 'doc_count' in bucket):
```

Comment:
Since we're no longer using 'terms' in the condition we should move
```
terms = aggregations[agg_name][agg_name]['buckets']
```
down to above this condition block.
```
if facet_type == 'exists':
```
To group the code a little better.